schema_transformations.py

```diff
@@ -722,6 +768,36 @@ def correctify_orcid(org_orcid):
     return "https://orcid.org/{}".format(orcs[-1])
 
 
+def correctify_scopus(org_scopus):
+    """Correct illformatted Scopus."""
+    # Get rid of all spaces.
+    new_scopus = org_scopus.replace(' ', '')
+
+    if not re.search("^\d{1,11}$", new_scopus):
+        # Return original value.
+        return org_scopus
+
+    return new_scopus
+
+
+def correctify_isni(org_isni):
+    """Correct ill-formatted ISNI."""
+    # Remove all spaces.
+    new_isni = org_isni.replace(' ', '')
+
+    # Upper-case X.
+    new_isni = new_isni.replace('x', 'X')
+
+    # The last part should hold a valid id like eg: 123412341234123X.
+    # If not, it is impossible to correct it to the valid isni format
+    new_isni = new_isni.split('/')
+    if not re.search("^[0-9]{15}[0-9X]$", new_isni[-1]):
+        # Return original value.
+        return org_isni
```

Comment:
Don't we need to return None in this case, so that calling functions know that the value can't be transformed, and they can log a warning message? Can we also capture the behaviour of this function in a unit test?

Reply:
Moved correctify functions to schema_transformation_utils and and added unit tests for them